homeassistant/components/sensor/steam_online.py

```diff
@@ -61,19 +65,28 @@ def state(self):
     # pylint: disable=no-member
     def update(self):
         """Update device state."""
-        self._profile = self._steamod.user.profile(self._account)
-        if self._profile.current_game[2] is None:
-            self._game = 'None'
-        else:
-            self._game = self._profile.current_game[2]
-        self._state = {
-            1: 'Online',
-            2: 'Busy',
-            3: 'Away',
-            4: 'Snooze',
-            5: 'Trade',
-            6: 'Play',
-        }.get(self._profile.status, 'Offline')
+        try:
+            self._profile = self._steamod.user.profile(self._account)
+            if self._profile.current_game[2] is None:
+                self._game = 'None'
+            else:
+                self._game = self._profile.current_game[2]
+            self._state = {
```

Comment:
Can you move that as constante on top of your script. That make it more readable.